.github/workflows/docker-push.yaml

```diff
@@ -19,76 +17,82 @@ jobs:
     permissions:
       contents: read
       packages: write
-      # This is used to complete the identity challenge
-      # with sigstore/fulcio when running outside of PRs.
       # id-token: write
 
     steps:
       - name: Check out the repo
         uses: actions/checkout@v4
 
-      # Install the cosign tool except for PR
-      # https://github.com/sigstore/cosign-installer
       # - name: Install cosign
       #   if: github.event_name != 'pull_request'
       #   uses: sigstore/cosign-installer@v3
 
-      # Set up BuildKit Docker container builder to be able to build
-      # multi-platform images and export cache
-      # https://github.com/docker/setup-buildx-action
+      - name: Set up QEMU
```

Comment:
Add a comment why you need QEMU in your case

Reply:
To help me build image for many CPU architectures without specify any in other steps

Reply:
Ok Add a comment in the yaml to explain what u said